dxr/lines.py

```diff
@@ -120,8 +120,12 @@ def ref_class(plugin, id):
                 warn('Ref subclass from plugin %s with ID %s was referenced '
                      'in the index but not found in the current '
                      'implementation. Ignored.' % (plugin, id))
-
-        payload = es_data['payload']
+         
```

Comment:
Let's fix [linter errors like this one](https://travis-ci.org/mozilla/dxr/builds/267092335#L2659-L2666) as we go so Travis will at least try to run the tests.